nucypher/cli/commands/stake.py

```diff
@@ -701,7 +703,7 @@ def prolong(general_config, transacting_staker_options, config_file, force, lock
     # Interactive
     if not lock_periods:
         stake_extension_range = click.IntRange(min=1, max=economics.maximum_allowed_locked, clamp=False)
-        max_extension = economics.maximum_allowed_locked - current_stake.periods_remaining
+        max_extension = economics.maximum_rewarded_periods - current_stake.periods_remaining
```

Comment:
it's better, but still is not fully correct, fixed in #1652

Reply:
Since it's in an another open PR - Shall I do anything here?

Reply:
I think it's ok to just leave it (if PR will be merged before) or discard - if merged after #1652